.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   test:
     parallelism: 4
     docker:
-      - image: electronuserland/builder:wine-mono
+      - image: electronuserland/builder:wine-mono-05.21
```

Comment:
Looks like we have a new failed test `targets` due to changed docker image :(

Reply:
>                     out={:timestamp=>"2021-05-12T09:36:17.960531+0200", :message=>"Process failed: /bin/sh failed (exit code 127). Full command was:[\"/bin/sh\", \"-c\", \"LANG=C bsdtar -czf .MTREE --format=mtree --options='!all,use-set,type,uid,gid,mode,time,size,md5,sha256,link' opt usr .PKGINFO .INSTALL\"]", :level=>:error}

`bsdtar` was removed from the base/Dockerfile in the latest PR. 

Not sure how to test that hypothesis